godot-core/src/builtin/color.rs

```diff
@@ -196,8 +198,6 @@ impl Color {
         self.a = from_u8(a);
     }
 
-    // TODO add getters and setters for h, s, v (needs generated property wrappers)
```

Comment:
I believe the `ColorHsv` addition removes the necessity of this TODO

Reply:
We would then move a bit away from providing direct accessors (like Godot) and enforce conversion to `ColorHsv`.

But I think it's fair in terms of having one approach to do things, so I'm fine with removing the TODO. Could you document in 1 sentence of the `struct Color` docs, that HSV space can be accessed using `ColorHsv::from()`? Otherwise this is hard to discover.